<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuessNumber</class>
 <widget class="QWidget" name="GuessNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuessNumber</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(62, 62, 62);</string>
  </property>
  <widget class="QPushButton" name="btn_guess">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(90, 199, 32);
border-color:  rgb(90, 199, 32);
border-style:solid;
border-width:3px;</string>
   </property>
   <property name="text">
    <string>Guess</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 100, 11);
border-style:solid;
border-width:3px;</string>
   </property>
   <property name="text">
    <string>Guess a number between 1-30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="nums">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_guess">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 100, 11);
border-style:solid;
border-width:3px;</string>
   </property>
   <property name="text">
    <string>5 guess left</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_score">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 100, 11);
border-style:solid;
border-width:3px;</string>
   </property>
   <property name="text">
    <string>Score : 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
